app/controllers/application_controller.rb

```diff
@@ -26,6 +26,17 @@ class UnprocessableEntity < StandardError
 
   private
 
+  def authenticate_user!
+    unless user_signed_in? ||
+             request.path.in?(
+               [new_user_session_path, destroy_user_session_path]
+             )
+      store_location_for(:user, request.fullpath) if request.path != start_path
+      flash[:info] = "You must be logged in to access this page"
```

Comment:
```suggestion
      flash[:info] = "You must be logged in to access this page."
```

Would we put a full stop here?